modules/local_storage/tests/modules/commerce_stock_local_test/config/install/commerce_stock.core_stock_events.yml

```diff
@@ -0,0 +1,3 @@
+core_stock_events_order_complete: true
+core_stock_events_order_cancel: true
+core_stock_events_order_updates: true
```

Comment:
@BBGuy don't misread, its the test module (not a modification of your implementation) I figured we should test if the config overwrite works.